source/resources.rst

```diff
@@ -8,13 +8,14 @@ For an in-depth introduction, we highly recommend
 Books We Like
 -------------
 * `Effective Computation in Physics <http://shop.oreilly.com/product/0636920033424.do>`_ (beginner)
+* `Python for Scientists <https://www.oreilly.com/programming/free/files/python-for-scientists.pdf>`_ (beginner)
 * `Python for Data Analysis <http://shop.oreilly.com/product/0636920023784.do>`_ (intermediate)
 * `Python and HDF5 <http://shop.oreilly.com/product/0636920030249.do>`_ (intermediate)
 
 Scientific Python Packages We Use
 ---------------------------------
 
-* `numpy tutorial <http://wiki.scipy.org/Tentative_NumPy_Tutorial>`_ -- core package for fast array computation in Python
```

Comment:
RIP "Tentative Numpy Tutorial", which taught me so much in grad school. :- D

Reply:
It's still possible to find it:
- https://scipy.github.io/old-wiki/pages/Tentative_NumPy_Tutorial
- http://www.cs.man.ac.uk/~barry/mydocs/MyCOMP28512/MS18_Notes/Python/Tentative_NumPy_Tutorial.pdf

but it can hardly be considered an official/formal source.